src/main/java/com/rtb/projectmanagementtool/user/UserData.java

```diff
@@ -24,6 +25,7 @@
 
   private long userID;
   private String AuthID; // this will be the ID from API
+  private String inviteCode; // code used to add user to a project
```

Comment:
Will the project admin (professor) be required to type these in to find students?  It would be unlikely for a professor to know these, unless if they have some other reference table to look at that would give the codes for students names.  In which case maybe we can provide that list (of users) for them?

Email would have been a better approach than inviteCode (with made up emails for now).